integration_tests/test_url_sandbox.py

```diff
@@ -19,7 +19,7 @@ class TestUrlSandbox(BaseIntegrationTest):
     expected_diff = '--- \n+++ \n@@ -1,4 +1,4 @@\n-import requests\n+from security import safe_requests\n \n-requests.get("www.google.com")\n+safe_requests.get("www.google.com")\n var = "hello"\n'
     expected_line_change = "3"
     change_description = UrlSandbox.CHANGE_DESCRIPTION
-    num_changed_files = 2
+    num_changed_files = 1
```

Comment:
This was happening because we previously didn't restrict the `--path-include` for these tests.

Reply:
Yeah you're probably right but now it's necessary in order for the other tests to work.